README.md

```diff
@@ -18,18 +115,64 @@ $ MLNX_OFED_VERSION=52220 /bin/bash build.sh
 ```
 Now the ovs-docker image created successfully
 
+## Pull docker image from docker hub 
+
+  - Install(https://docs.docker.com/engine/install/centos/) and start docker:
+    ```
+    service docker start
+    ```
+  - Container from docker.io/mellanox/ovs-forwarder can be used:
+    Check at the following link  https://hub.docker.com/r/mellanox/ovs-forwarder/tags
+    ```
+    docker pull mellanox/ovs-forwarder:52220
+    ```
+  - check image pulled successfully
+    ```
+    docker images
+    ```
+  - clone containerized-ovs-forwarder:
+    ```
+    git clone https://github.com/Mellanox/containerized-ovs-forwarder
```

Comment:
we should do 
git checkout origion/v5.2 -b v5.2 (one I create a branch)

Reply:
will be added after generating the branch